src/components/Layout/GreetingPageLayout/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from 'styled-system/css';
+import { vstack } from 'styled-system/patterns';
+
+const container = vstack({
+  m: '0 auto',
+  h: '100%',
+  alignItems: 'center',
+  p: '1.6rem',
+  height: 'calc(100vh - 6rem)',
+});
+
+const headTitle = css({
+  textStyle: 'title1',
+  color: 'text.100',
+  textAlign: 'center',
+  mb: '1rem',
+  '& span': {
+    color: 'main.blue',
+  },
+});
+
+const subTitle = css({
+  color: 'text.200',
+  textAlign: 'center',
+  mb: '1.5rem',
+});
+
+const positionedParent = css({ pos: 'relative' });
+
+const positionedWrapper = css({
+  position: 'absolute',
+  top: '227px',
```

Comment:
```suggestion
  top: '22.7rem',
```
px -> rem으로 수정해야할 것 같습니다